i18n/es/docusaurus-plugin-content-docs/current/testnet/Faucet.tsx

```diff
@@ -86,13 +86,12 @@ export const Faucet = () => {
 
       <div className="faucetResultContainer">
         {hash && (
-          <>
-            <p>Sus fondos han sido enviados!</p>
-            <p className="sm">
-              <a href={`https://rococo.subscan.io/extrinsic/${hash}`} target="_blank">Haga clic aquí</a>{' '}
-              para ver la transacción.
-            </p>
-          </>
+          <button className="faucet-success w-full border-none" onClick={() => window.open(`https://rococo.subscan.io/extrinsic/${hash}`, '_blank')}>
+            <div className="faucet-success-ic">
+              <p className="faucet-success-msg">Sus fondos han sido enviados!</p>
```

Comment:
at some point we should use translation config files vs copying React components. Not a big deal now...